### DISCUSSION TITLE
null is not an object ( evaluating 'RNGestureHandlerModule.default.createGestureHandler')

### DISCUSSION THREAD

**Discussion by Akash-T2S** [software-mansion/react-native-gesture-handler#1472] [General]
null is not an object ( evaluating 'RNGestureHandlerModule.default.createGestureHandler)

Im trying to use react-native-gesture-handler in macos using react-native-macos

"react": "17.0.1",
"react-native": "0.64.1",
"react-native-gesture-handler": "1.10.3",
"react-native-macos": "0.63.0-0",

**Comment by jgonet** [comment]
We don't support react-native-macos right now.
